spec/errorSpec.js

```diff
@@ -1,6 +1,5 @@
 var webdriver = require('selenium-webdriver');
 var common = require('./common.js');
-require('../index.js');
 
```

Comment:
That's a good point. I don't think it's necessary, but I bet there's an edge case where that could bite us.